frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -73,18 +73,53 @@ export default function ChangeAddress() {
   };
 
   const errorMessages = {
-    homeAddress: actionData?.errors.homeAddress?._errors[0],
-    mailingAddress: actionData?.errors.mailingAddress?._errors[0],
+    homeAddress: actionData?.errors.fieldErrors.homeAddress?.[0],
+    mailingAddress: actionData?.errors.fieldErrors.mailingAddress?.[0],
   };
 
+  /**
+   * Gets an error message based on the provided internationalization (i18n) key.
+   *
+   * @param errorI18nKey - The i18n key for the error message.
+   * @returns The corresponding error message, or undefined if no key is provided.
+   */
+  function getErrorMessage(errorI18nKey?: string): string | undefined {
+    console.log('hi');
+    console.log(errorI18nKey);
```

Comment:
Remove those `console.log()`

Reply:
removed in commit https://github.com/DTS-STN/canadian-dental-care-plan/pull/214/commits/d194f0f07d8082bb385484f22c5cfe7967b87e06